docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -102,12 +120,27 @@ az account set --subscription "My Demos"
 
 # change the active subscription using the subscription ID
 az account set --subscription "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+```
+
+---
+ # [Bash](#tab/bash)
```

Comment:
Before we start a tab group, we need to tell the reader what we're doing, like "You can also change your subscription using a variable. Here is an example:."

Reply:
Done